tests/src/test/330/scala/com/nvidia/spark/rapids/IntervalSuite.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids
+
+import java.io.File
+import java.time.{Duration, Period}
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+/**
+ * Can not put this suite to cache_test.py because of currently Pyspark not have year-month type.
+ * See: https://github.com/apache/spark/blob/branch-3.3/python/pyspark/sql/types.py
+ */
+class IntervalSuite extends SparkQueryCompareTestSuite {
```

Comment:
Need to skip these tests for Spark before 3.3.0

Reply:
It's in `test/330` directory and will skip for Spark before 3.3.0